hudi-sync/hudi-sync-common/src/main/java/org/apache/hudi/sync/common/AbstractSyncHoodieClient.java

```diff
@@ -75,6 +76,8 @@ public abstract void createTable(String tableName, MessageType storageSchema,
 
   public abstract void updatePartitionsToTable(String tableName, List<String> changedPartitions);
 
+  public abstract void updateTableProperties(String tableName, Map<String, String> tableProperties);
```

Comment:
since this is abstract class and not every implementation will have some concrete override, can we make this empty here so that HoodieDLAClient does not need to do a no op override.